packages/integration-tests/index.js

```diff
@@ -69,11 +69,13 @@ async function waitForCompletedExecution(executionArn) {
   let statusCheckCount = 0;
 
   // While execution is running, check status on a time interval
+  /* eslint-disable no-await-in-loop */
```

Comment:
nit: I usually prefer to add the eslint-disable comment on the same line that is causing the error.  So, something like this:

`await timeout(waitPeriodMs); // eslint-disable-line no-await-in-loop`